manual/parts/threads.tex

```diff
@@ -211,7 +211,7 @@ \subsection{Scheduling Context Creation} \label{sec:sc_creation}
 time.  Threads run on the node that their scheduling context is configured for.  Scheduling context
 parameters can then be set and updated using
 \apifunc{seL4\_SchedControl\_ConfigureFlags}{schedcontrol_configureflags}, which allows the budget and period
-to be specified along with a bitwise OR'd set of the following flags.
+to be specified along with a bitwise OR'd set of the following flags:
```

Comment:
I don't know how consistent we are in the manual with that, but journal styles would usually not put a colon here, but a full stop as in the original.

```suggestion
to be specified along with a bitwise OR'd set of the following flags.
```